rotkehlchen/globaldb/handler.py

```diff
@@ -106,13 +174,15 @@ class GlobalDBHandler():
     """A singleton class controlling the global DB"""
     __instance: Optional['GlobalDBHandler'] = None
     _data_directory: Optional[Path] = None
+    _msg_aggregator: Optional['MessagesAggregator'] = None
     _packaged_db_conn: Optional[DBConnection] = None
     conn: DBConnection
 
     def __new__(
             cls,
             data_dir: Optional[Path] = None,
             sql_vm_instructions_cb: Optional[int] = None,
+            msg_aggregator: Optional[MessagesAggregator] = None,
```

Comment:
makes no sense. Leave that out. Message aggregator should not be an argument to the GlobalDB. Remove it.